src/pages/MetricDetail.svelte

```diff
@@ -197,6 +225,15 @@
         {/each}
       </td>
     </tr>
+    {#if glamUrl}
+      <tr>
+        <td>GLAM</td>
+        <td>
+          <a
+            href={glamUrl(metric.bigquery_names.glam_etl_name)}>{glamUrl(metric.bigquery_names.glam_etl_name)}</a>
```

Comment:
Showing the full URL in the UI seems a little verbose (you can probably skip the `const` declaration above if you take it out, as well).

```suggestion
            href={glamUrl(metric.bigquery_names.glam_etl_name)}>{metric.bigquery_names.glam_etl_name}</a>
```

Reply:
I updated this but still need the `const` for the `{#if}` check to show the table row.